recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -190,6 +198,14 @@ def _patch_sources(self):
                 os.path.join(self.generators_folder, "SPIRV-ToolsConfig.cmake"),
                 os.path.join(self.generators_folder, "SPIRV-Tools-optConfig.cmake"),
             )
+        if self.settings.os == "Android":
+            # INFO: libVkLayer_utils.a: error: undefined symbol: __android_log_print
+            # https://github.com/KhronosGroup/Vulkan-ValidationLayers/commit/a26638ae9fdd8c40b56d4c7b72859a5b9a0952c9
+            replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
+                        "VkLayer_utils PUBLIC Vulkan::Headers", "VkLayer_utils PUBLIC Vulkan::Headers -landroid -llog")
```

Comment:
Probably needs
```python
self.cpp_info.system_libs.extend(["android", "log"])
```
as well?

Reply:
you're right, this is the way to fix the issue @perseoGI

Reply:
It is still needed that replace_in_file as we saw, the issue was on the linkage stage, not during the consumption of the library.